### PR TITLE
Portifolio footer

### PR DESCRIPTION
**Pull request by sangwa7**
### -Added Footer Section with two sections:

-Footer Header 
   
-Contact Form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,149 +1,285 @@
...
+    </section>
+    <footer class="footer">
+      <div class="footer-header">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </div>
+      <div class="footer-form">
+        <form method="post" action="https://formspree.io/f/1">
```
Comment (AnselemOdims) [inline index.html]: - [ ] Please ensure that you have registered properly and gotten a valid ID on formspree. At the moment, the form is not been submitted.

File: index.html [hunk 2]
```diff
@@ -1,149 +1,285 @@
...
+          <textarea
+            name="user-message"
+            id="user-message"
+            cols="30"
+            rows="10"
+            maxlength="500"
+            required
+            placeholder="Enter text here"
+          ></textarea>
+          <button type="submit" class="see-project-button">Get in touch</button>
```
Comment (AnselemOdims) [inline index.html:251]: - [ ] Kindly ensure that the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. At the moment, only hover matches the design.

image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @sangwa7, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!. You can tag me using @AnselemOdims for a quicker response.

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by elmar8287 — APPROVED** [review]
Hi @sangwa7, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**